res/controllers/common-controller-scripts.js

```diff
@@ -315,8 +315,10 @@ script.spinback = function(channel, control, value, status, group) {
    Output:  none
    -------- ------------------------------------------------------ */
 script.brake = function(channel, control, value, status, group) {
-    // disable on note-off or zero value note/cc
-    engine.brake(parseInt(group.substring(8,9)), ((status & 0xF0) != 0x80 && value > 0));
+	// calculate current playback speed
+	var currentRate = engine.getValue(group,"bpm") / engine.getValue(group,"file_bpm");
```

Comment:
Okay, I was asking in case it would be easier to just get the `rate` value without having to manipulate it at all. I'm okay with the naming as is.